examples/deadline/All-In-AWS-Infrastructure-Basic/python/README.md

```diff
@@ -32,8 +32,16 @@ These instructions assume that your working directory is `examples/deadline/All-
     popd
     pip install ../../../../dist/python/aws-rfdk-<version>.tar.gz
     ```
-4.  Change the value in the `deadline_client_linux_ami_map` variable in `package/config.py` to include the region + AMI ID mapping of your EC2 AMI(s) with Deadline Worker.
+4.  Change the value in the `deadline_client_linux_ami_map` variable in `package/config.py` to include the region + AMI ID mapping of your EC2 AMI(s) with Deadline Worker. You can use the follow AWS CLI query to find AMI ID's:
```

Comment:
Nit: should it be 
```suggestion
You can use the following AWS CLI query to find AMI ID's:
```
I might be wrong :) No need to fix it.

Reply:
You are correct. I've fixed it.